example/CMakeLists.txt

```
@@ -0,0 +1,49 @@
+# Conditionally adding new sources to the main T8 target as in the original BS looked wrong...
+add_library( T8ex )
```

```suggestion
add_library( t8example )
```

Can you rename this to `libt8example.so`? Than it is much clearer what this library is for.